SU-SOE/stanford_soe_helper
# 7.x 2.x refine dm masthead

# READY FOR REVIEW

# Summary

- Fixes the masthead jump back to big size. Now it stays petit.
- Brings hamburger menu into alignment with masthead.

# Needed By (Date)
- Sprint end

# Criticality
- Fixes broken stuff

# Steps to Test
- Switch to this branch
- May need to revert feature
- Navigate to /magazine, /magazine/issue, and any article page
- Verify you see a small masthead and the hamburger menu in alignment.

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2281
## Related PRs

## More Information
See screenshots from ticket:

https://stanfordits.atlassian.net/secure/attachment/37257/Screen%20Shot%202017-10-12%20at%2017.13.12.png
https://stanfordits.atlassian.net/secure/attachment/37258/Screen%20Shot%202017-10-12%20at%2017.13.40.png

## Folks to notify
@kbrownell 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
